ambry-network/src/main/java/com.github.ambry.network/NetworkMetrics.java

```diff
@@ -30,23 +31,30 @@
 public class NetworkMetrics {
```

Comment:
fine. I will make changes and push in all in one go (along with addressing other feedbacks)